romancal/photom/photom.py

```diff
@@ -0,0 +1,169 @@
+import logging
+import warnings
+
+import numpy as np
+from astropy import units as u
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+# Conversion factor from MJy/sr to uJy/arcsec^2
+MJSR_TO_UJA2 = (u.megajansky / u.steradian).to(u.microjansky / u.arcsecond / u.arcsecond)
+
+# Conversion factor from square arcseconds to steradians
+A2_TO_SR = (np.pi / (180. * 3600.))**2
+
+
+def photom_io(input_model, photom_metadata):
+    """
+    Short Summary
+    -------------
+    Combine photometric scalar conversion factors and add to the science metadata.
+
+    Parameters
+    ----------
+    input_model : Roman level 2 image datamodel
+        input roman datamodel
+
+    photom_metadata : list
+        Set of matched photom meta data keys
+
+    Returns
+    -------
+
+    """
+    # Get the scalar conversion factor.
+    conversion = photom_metadata['photmjsr']  # unit is MJy / sr
```

Comment:
It looks like `photom_metadata` here is a dict, while it's defined as a list in the docstring.

Reply:
Good catch - changed.